rntape/app.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "rntape",
+  "displayName": "rntape"
+}
```

Comment:
Could you elaborate on what you mean by this? 😅

Reply:
The end of the file was without a last line. Some automated tools may add the last line and some editors do that too. This will end up as noise in pull requests which is why i am in the habit of simply adding a line at the end of a file whenever I notice it missing

Reply:
How about adding `.editorconfig` with the default configuration?
For me this usually helps to solve this issue.

Reply:
the `app.json` was generated automatically when setting up this and as such an `.editorconfig` would not have solved _this_ particular issue, but yes: we can add that.